Jenkinsfile.github

```diff
@@ -87,7 +87,9 @@ pipeline {
                 stage("Charts and Images") {
                     steps {
                         withCredentials([usernamePassword(credentialsId: credentialsId, usernameVariable: 'ARTIFACTORY_USER', passwordVariable: 'ARTIFACTORY_TOKEN')]) {
+                            // sleep for 1 minute to avoid 'context deadline exceeded' cuased by API documentation
                             sh """
+                                sleep 60
```

Comment:
We'll have to run API generator and other steps on the same VM, since we need to download and use `docs-csm-*.rpm` file. Let's introduce 60 seconds delay at the beginning of generator run. Pulling and running too many containers at once causes `context deadline exceeded` errors.